opensearch-k8s-operator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: opensearch-k8s-operator
+  version: 2.6.1
+  epoch: 0
+  description: OpenSearch Kubernetes Operator
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
```

Comment:
can be removed as well because looking at [upstream deployment](https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/opensearch-operator/Dockerfile) it seems a single binary would be enough for the controller run. They don't copy certificates in the final stage.

Reply:
Thank you. I'll drop `ca-certificates-bundle`.

Reply:
Removed.